<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jnlPrincipal</class>
 <widget class="QMainWindow" name="jnlPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ninterplayer | Nintersoft</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="conteudo">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="layout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionAbrir_arquivo_de_udio"/>
    <addaction name="actionAbrir_arquivo_de_v_deo"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <widget class="QMenu" name="menuFerramentas">
    <property name="title">
     <string>Ferramentas</string>
    </property>
    <addaction name="actionConfigura_es"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuFerramentas"/>
   <addaction name="menuAjuda"/>
  </widget>
  <action name="actionAbrir_arquivo_de_v_deo">
   <property name="text">
    <string>Abrir arquivo de vídeo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionAbrir_arquivo_de_udio">
   <property name="text">
    <string>Abrir arquivo de áudio</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionConfigura_es">
   <property name="text">
    <string>Configurações</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
